build-farm/platform-specific-configurations/aix.sh

```diff
@@ -127,7 +127,7 @@ else
   export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} DF=/usr/sysv/bin/df"
 fi
 
-if [ "$JAVA_FEATURE_VERSION" -ge 11 ]; then
+if [ "$JAVA_FEATURE_VERSION" -le 21 ]; then
```

Comment:
We still build JDK8 with XLC13, so this should probably be something like:
   `if [ "$JAVA_FEATURE_VERSION" -le 21 -a "$JAVA_FEATURE_VERSION" -ge 11 ]; then`